hub.py

```diff
@@ -338,6 +338,15 @@ def apply_hub_template_fixes(self, generated_config, proxy_secret_key):
 
         }
 
+        # Generate a token for the docs service
+        docs_token = hmac.new(proxy_secret_key, f'docs-{self.spec["name"]}'.encode(), hashlib.sha256).hexdigest()
+        # Register the docs service
+        generated_config['jupyterhub']['hub']['services']['docs'] = {
+            'apiToken': docs_token,
```

Comment:
Does the docs service need an apiToken? It isn't making any API calls to the hub, so maybe not necessary. I'm not sure if you can run a service without explicitly setting an apitoken though.

Reply:
It turns out, it didn't need an api token :D Thanks for suggesting to try this <3!